Makefile

```diff
@@ -0,0 +1,36 @@
+export GO111MODULE := on
+DIST_DIR ?= ${PWD}/bin
+HOME := ${HOME}
+PROJECT_NAME := terraform-provider-spinnaker
+
+.PHONY: help
+help:  ## Show help messages for make targets
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: setup
+setup: ## Download tools required to check and build
+ifeq ($(shell uname -s), Darwin)
+	@brew install golangci-lint
+else
+	@curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh
+endif
+
+.PHONY: build
+build: ## Download dependencies and compile project
+	@CGO_ENABLED=0 go build -o ${DIST_DIR}/${PROJECT_NAME} main.go
+
+.PHONY: install
+install: build ## Builds and installs the terraform provider
```

Comment:
would be nice if we could have a make that installs the latest provider in makefile of infraregistry

Reply:
We have it there but pinned.